schemas/credentials.json

```diff
@@ -357,23 +357,56 @@
                 "description": "SD-WAN Portal Host password to login into command line",
                 "default": "",
                 "encrypt": true,
+                "sectionEnd": "Portal",
                 "propertyOrder": 360
             },
             "smtp_auth_username": {
                 "type": "string",
                 "title": "SMTP username",
                 "description": "Username for SMTP authentication",
                 "default": "",
+                "sectionBegin": "SMTP",
                 "propertyOrder": 370
             },
             "smtp_auth_password": {
                 "type": "string",
                 "title": "SMTP password",
                 "description": "Password for SMTP authentication",
                 "default": "",
-                "sectionEnd": "Portal",
+                "sectionEnd": "SMTP",
                 "propertyOrder": 380
-           }
+           },
+           "netconf_vm_username": {
+                "type": "string",
+                "title": "Netconf Manager VM username",
+                "default": "root",
+                "description": "Username for Netconf Manager VM. Default user is root",
+                "sectionBegin": "Netconf Manager",
+                "propertyOrder": 390
+            },
+            "netconf_vm_password": {
+                "type": "string",
+                "title": "Netconf Manager VM password for running sudo commands",
+                "description": "Password for Netconf manager VM",
+                "default": "",
+                "propertyOrder": 400
+            },
+            "netconf_username": {
```

Comment:
If this is really a username specific for a certificate? If so, the property name should be `netconf_certificate_username`. My assertion assumes, of course, that the installer will know what a certificate username is. If the user knows this by some other name, that other name should be used. This generally applies also to the password property in line 401.

Reply:
This is the netconf username, i should remove certificate from the title to remove the confusion. The docs refer to it as the netconf username.